HenHen.Core/Worlds/Plants/FlowerBreed.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Affectionate Dove <contact@affectionatedove.com>.
+// Licensed under the Affectionate Dove Limited Code Viewing License.
+// See the LICENSE file in the repository root for full license text.
+
+using System.Collections.Generic;
+
+namespace HenHen.Core.Worlds.Plants
+{
+    public record FlowerBreed(string Name,
+        IReadOnlyList<int> GrowthStagesDuration,
+        IReadOnlyList<int> GrowthStagesDurationVariance,
```

Comment:
Replace `GrowthStagesDuration` and `GrowthStagesDurationVariance` with RandomIntRange (see #60).